test/unit/controllers/HeaderValidatorSpec.scala

```diff
@@ -16,8 +16,8 @@
 
 package unit.controllers
 
-import org.scalatestplus.mockito.MockitoSugar
 import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatestplus.mockito.MockitoSugar
```

Comment:
revert import change

Reply:
needs mockitoSugar to mock notificatiuon logger